content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -43,10 +55,83 @@ To start a test, click the :guilabel:`Run` button, represented by a :guilabel:`
 icon beside the first activity in the workflow. When clicked, the page reloads, and Odoo shows the
 various results (and analytics) connected to that specific activity as they occur, in real-time.
 
+.. note::
+   If a child activity is schedule beneath a parent activity, that child activity will be revealed
+   in the workflow, once that parent activity has been run, via the :guilabel:`▶️ (play button)`
+   icon.
+
 .. image:: testing_running/workflow-test-progress.png
    :align: center
    :alt: Workflow test progress in Odoo Marketing Automation.
 
-Once all the workflow activities are completed, the test will end, and move to the
-:guilabel:`Completed` stage. To stop a test before all the workflow activities are completed, click
-the :guilabel:`Stop` button in the upper-left corner of the campaign test page.
+Once all the workflow activities are completed, the test ends, and the status bar (in the
+upper-right corner) moves to the :guilabel:`Completed` stage.
+
+To stop a test before all the workflow activities are completed, click the :guilabel:`Stop` button
+in the upper-left corner of the campaign test page.
+
+Running campaigns
+=================
+
+To run a campaign, navigate to :menuselection:`Marketing Automation app`, and select the desired
+campaign to run.
```

Comment:
```suggestion
To run a campaign, navigate to :menuselection:`Marketing Automation app`, and select the desired
campaign to run.
```

You mention Marketing Automation Application at the start of the doc and now use a menu selection for just app here. May want to keep consistent.

Reply:
Great catch!